src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key of Object.keys(robot)) {
+    const a = robot[key];
```

Comment:
use more meanfull variable names

Reply:
thank you for the correction
I'll keep it in mind :)